src/data/markdown/docs/02 javascript api/06 k6-execution.md

```diff
@@ -71,7 +71,7 @@ Control the test execution.
 
 | Property | Type | Description |
 |----------|------|-------------|
-| abort | function | It immediately aborts a k6 load test run with the exit code `108`. Aborting a test is fully supported in `k6 cloud`, preserving the same user experience for both local and cloud users. |
+| abort | function | It immediately aborts a k6 load test run with the exit code `108`. |
```

Comment:
```suggestion
| abort | function | It ~~immediately~~ aborts a k6 load test run with the exit code `108`. |
```
@imiric Is it immediately the right word? Can be considered immediately also for the cloud?

Reply:
It's not quite "immediate" in the cloud as the processing winds down, but it might be worth mentioning here that `teardown()` will still run if `abort()` is called. So in that sense, I would remove "immediately" from here.